controllers/alertsconfig_controller.go

```diff
@@ -135,7 +136,10 @@ func (r *AlertsConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		}
 		var alert wf.Alert
 		//Get the processed wf alert
-		if err := controllercommon.GetProcessedWFAlert(ctx, &wfAlert, &config, &alert); err != nil {
+		//merge the alerts config global params and individual params
+		params := utils.MergeMaps(ctx, config.Params, globalMap)
```

Comment:
Should individual params take the privilege here? the first map is baseParams and second one is overwriteParams, so here globalMap overwrites individual params

Reply:
Yeah- You are right!!